### PR TITLE
Update code-splitting-css.md

### PR DESCRIPTION
**Pull request by yhpark**
Fixed broken url and typos.

### CODE REVIEW

File: content/guides/code-splitting-css.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ contributors:
 
 In webpack, when you use the css-loader and import CSS into your JavaScript files, the CSS is bundled along with your JavaScript.
 This has the disadvantage that, you will not be able to utilize the browser's ability to load CSS asynchronously and parallel. Instead, your page will have to wait until your whole JavaScript bundle is loaded, to style itself.
-webpack can help with this problem by bundling the CSS separately using [extract-text-webpack-plugin](https://github.com/webpack/extract-text-webpack-plugin) and the [css-loader](https://github.com/webpack/css-loader).
+Webpack can help with this problem by bundling the CSS separately using [extract-text-webpack-plugin](https://github.com/webpack/extract-text-webpack-plugin) and the [css-loader](https://github.com/webpack/css-loader).
```
Comment (bebraw) [inline content/guides/code-splitting-css.md]: It should be just "webpack" in these docs.

Comment (yhpark) [inline content/guides/code-splitting-css.md]: It's at the beginning of a sentence.. so should be capital letter regardless.

Comment (SpaceK33z) [inline content/guides/code-splitting-css.md]: https://github.com/webpack/media#name

Comment (yhpark) [inline content/guides/code-splitting-css.md]: Fixed and amended the commit..

**Pull request merged**